lib/src/view/game/game_loading_board.dart

```diff
@@ -271,7 +271,7 @@ class _AnimatedLobbyNumberState extends State<_AnimatedLobbyNumber> {
         end: value,
       ),
       curve: Curves.linear,
-      duration: const Duration(seconds: 3),
+      duration: const Duration(seconds: 6),
```

Comment:
I'm really not sure it fixes the issue. Were you able to reproduce it?

Reply:
I can see that the counter changes less frequently when I open/close many tabs with lichess at the same time. I don't know if it's possible to simulate the sheer amount of people opening and closing lichess at infrequent intervals, like what the app would observe in prod. Still, it shouldn't make a difference because there are never multiple transitions happening at the same time, since the widget is rebuilt every time the state changes.